.all-contributorsrc

```diff
@@ -3297,6 +3297,12 @@
       ]
     },
     {
+      "login": "marcosfelt",
+      "name": "Kobi Felton",
+      "avatar_url": "https://avatars.githubusercontent.com/u/25933639?v=4",
+      "profile": "https://github.com/marcosfelt",
+      "contributions": [
+        "code"
```

Comment:
I think your new additions here are not in right format, and may mess up both yours and Shounak's badges. Can you please fix it, and add as an independent block?

Reply:
Fixed! Thanks for noticing this @yarnabrina!